app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsRobot.kt

```diff
@@ -122,6 +144,10 @@ class SettingsRobot {
             return SettingsSubMenuEnhancedTrackingProtectionRobot.Transition()
         }
     }
+
+    companion object {
+        const val GOOGLE_PLAY_SERVICES_PACKAGE_NAME = "com.android.vending"
+    }
```

Comment:
This is fine for now but I'm wondering if we should start a `Constant.kt` and make an 
```Kotlin
object Constants { 
 const val PACKAGES = arrayOf { 
     "com.android.vending"
     "other packages here" 
} 
```` 
alongside any other new constants

Reply:
Some of these will definitely be reused so it makes sense to me 👍